src/platform/packages/shared/response-ops/alerts-table/types.ts

```diff
@@ -109,6 +110,7 @@ export interface CasesService {
   helpers: {
     groupAlertsByRule: (items: any[]) => any[];
     canUseCases: (owners: Array<'securitySolution' | 'observability' | 'cases'>) => any;
+    getRuleIdFromEvent: typeof getRuleIdFromEvent;
```

Comment:
It seems that the `getRuleIdFromEvent` function is introduced only to determine a type. The alerts table does not use that code. Is the type needed? Did TS produce an error?

Reply:
> Change the cases?: CasesService; to cases?: CasesPublicStart; in src/platform/packages/shared/response-ops/alerts-table/types.ts with import import type { CasesPublicStart } from '@kbn/cases-plugin/public';. This may create circular dependencies, and TS will produce an error

I can confirm that's the reason why I used the custom type